cpp/ql/lib/semmle/code/cpp/controlflow/internal/ConstantExprs.qll

```diff
@@ -122,7 +122,7 @@ private predicate isFunction(@element el) {
  */
 pragma[noopt]
 private predicate callHasNoTarget(@funbindexpr fc) {
-  exists(@function f |
+  exists(Function f |
```

Comment:
Should be kept.

Reply:
Fixed in fe2c806c2db5f3dceaf2877f34a25fcbce34337b